docker/README.md

```diff
@@ -0,0 +1,144 @@
+# FIWARE IoT Agent for SigFox
+
+[![FIWARE IoT Agents](https://nexus.lab.fiware.org/repository/raw/public/badges/chapters/iot-agents.svg)](https://www.fiware.org/developers/catalogue/)
+[![](https://nexus.lab.fiware.org/repository/raw/public/badges/stackoverflow/iot-agents.svg)](https://stackoverflow.com/questions/tagged/fiware+iot)
+
+This IoT Agent is designed to be a bridge between the [Sigfox](http://www.sigfox.com/en/) callbacks protocol and the OMA
+NGSI protocol used by the [Orion Context Broker](https://github.com/telefonicaid/fiware-orion) as well as by other
+components of the FIWARE ecosystem.
+
+It is based on the [IoT Agent Node.js Library](https://github.com/telefonicaid/iotagent-node-lib). Further general
+information about the FIWARE IoT Agents framework, its architecture and the common interaction model can be found in the
+library's GitHub repository.
+
+This project is part of [FIWARE](https://www.fiware.org/). For more information check the FIWARE Catalogue entry for the
+[IoT Agents](https://github.com/Fiware/catalogue/tree/master/iot-agents).
+
+## How to use this image
+
+The IoT Agent must be instantiated and connected to an instance of the
+[Orion Context Broker](https://fiware-orion.readthedocs.io/en/latest/), a sample `docker-compose` file can be found
+below.
+
+If the `IOTA_REGISTRY_TYPE=mongodb`, a [MongoDB](https://www.mongodb.com/) database instance is also required - the
+example below assumes that you have a `/data` directory in your hosting system in order to hold database files - please
+amend the attached volume to suit your own configuration.
+
+```yml
+version: "3.1"
+
+volumes:
+    mongodb: ~
+
+services:
+    iot-agent:
+        image: fiware/sigfox-iotagent
+        hostname: iot-agent
+        container_name: fiware-iot-agent
+        depends_on:
+            - mongodb
+        expose:
+            - "4041"
+            - "7896"
+        ports:
+            - "4041:4041"
+            - "7896:7896"
+        environment:
+            - "IOTA_CB_HOST=orion"
```

Comment:
Do you have checked all these env vars exists also for IOTA Sigfox? Specially with regards to the southbound part (e.g. not sure if in this case we have a IOTA_HTTP port or Sigfox implies a fixed port).

In addition, for env vars that exists, the default value should match the one used by this agent. For instance, I'm not sure that 4041 is the right default for Sigfox (I may be wrong, anyway).

Reply:
We have been checking on the code and it seems that appart from the env vars defined in IOTA Agents Library, the IOTA Sigfox implements the following variable: IOTA_SIGFOX_PORT (from https://github.com/telefonicaid/sigfox-iotagent/blob/master/lib/configService.js#L41).

CC: @AlvaroVega

Reply:
`ENV` altered to use and expose the SIGFOX port. Fixed efc68c0